packages/nextjs/pages/index.tsx

```diff
@@ -20,6 +35,20 @@ const Home: NextPage = () => {
             <span className="block text-4xl font-bold">Batch 1</span>
           </h1>
           <p className="text-center text-lg">Get started by taking a look at your batch GitHub repository.</p>
+          <p className="text-lg flex gap-2 justify-center">
+            <span className="font-bold">Connected wallet status:</span>
+            <span>{allowList?.toString()}</span>
+          </p>
+          <p className="text-lg flex gap-2 justify-center">
+            <span className="font-bold">Connected wallet check-in status:</span>
+            <span>
+              {isAllowListLoading
```

Comment:
It might be more fitting to use the `isLoading` value from the `yourContractAddress` read here instead of `isAllowListLoading`

Reply:
Actually, I'm not really sure why, but `isAllowListisLoading` is never true for when testing locally. My understanding is that `isLoading` should be `true` on the very first call, before there is any data cached, and then `false` for any fetch after that (which sets `isFetching` to `true`). 

In any case, maybe you could use `isFetching` instead to get the "Loading..." condition to work?

Reply:
Thank you @swellander actually I used isAllowListLoading feature as you mentioned above and expected it served the same purpose with isCounterLoading. Maybe isFetching is a good candidate here too. Anyway this issue already merged in another PR so I'm closing this one. Just wanted to test myself, thanks your reivew again :)